deploy/operator.yaml

```diff
@@ -17,20 +17,24 @@ spec:
         - name: local-storage-operator
           image: quay.io/openshift/origin-local-storage-operator
           ports:
-          - containerPort: 60000
-            name: metrics
+            - containerPort: 60000
+              name: metrics
           command:
-          - local-storage-operator
+            - local-storage-operator
           imagePullPolicy: Always
           env:
             - name: WATCH_NAMESPACE
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
             - name: OPERATOR_NAME
-              value: "local-storage-operator"
+              value: local-storage-operator
             - name: OPERATOR_NAME
-              value: "local-storage-operator"
+              value: local-storage-operator
```

Comment:
Was there anything wrong with existing yaml here?

Reply:
It got changed by the operator-sdk generated file.

Reply:
Again - this is not required (you can wipe away operator-sdk modifications). in-fact we are trying to move away from these files in `deploy` directory so as we can have unified yamls between CSV and YAML that is used for testing.

Reply:
reverted backk